EXRPdfReader/Views/PdfTableViewController.m

```diff
@@ -0,0 +1,133 @@
+//
+//  PdfTableViewController.m
+//  EXRPdfReader
+//
+//  Created by Arnold Joseph Caesar Esteban on 12/11/20.
+//  Copyright © 2020 Arnold Joseph Caesar Esteban. All rights reserved.
+//
+
+#import "PdfTableViewController.h"
+#import "PdfContentViewController.h"
+#import "PdfMetadata.h"
+#import "EXRPdfMetadataXMLParserDelegate.h"
+#import "PdfTableViewCell.h"
+#import "PdfRootViewController.h"
+
+static NSString *const kCellIdentifier = @"pdfTableViewCell";
+
+@interface PdfTableViewController ()
+
+@property (strong, nonatomic) NSArray<PdfMetadata *> *pdfs;
+
+@end
+
+@implementation PdfTableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    [self setupCellHeight];
+    [self setupAvailablePdfs];
+}
+
+#pragma mark - Table view data source
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
+    return 1;
+}
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
+    return self.pdfs.count;
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
+    PdfTableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:kCellIdentifier];
+    if (!cell) {
+        [tableView registerNib:[UINib nibWithNibName:@"PdfTableViewCell"
+                                              bundle:[NSBundle mainBundle]]
+        forCellReuseIdentifier:kCellIdentifier];
+        cell = [tableView dequeueReusableCellWithIdentifier:kCellIdentifier];
+    }
+    if (indexPath.item >= self.pdfs.count) {
```

Comment:
Best to check this first thing in the method, then handle (early return)

Assertion is good 🍇